src/main/java/org/cnu/realcoding/controller/DogController.java

```diff
@@ -25,5 +25,18 @@ public Dog getDogByAllKey(@PathVariable String name,
         return dogManagementService.getDogByAllKey(name, ownerName, ownerPhoneNumber);
     }
 
+    @GetMapping("/dogs/{ownerPhoneNumber")
```

Comment:
}

Reply:
why do you use RequestParam? in line 29

Reply:
I found only RequestParam properly works.